README.md

```diff
@@ -0,0 +1,28 @@
+# Flask-ping
+Blueprint for a `/ping` endpoint that can be added to python [Flask](https://flask.palletsprojects.com/en/1.1.x/) applications.
+
+## Usage
+To add the `/ping` endpoint to your application add the following:
+```python
+from flask_ping import get_ping_blueprint
+
+app.register_blueprint(get_ping_blueprint())
+```
```

Comment:
minimal but effective documentation for integrating the library into a Flask app.

Related to documentation: https://github.com/libero/flask-ping/blob/master/LICENSE should be changed to https://github.com/libero/browser/blob/master/LICENSE as the copyright should be assigned to eLife